app/views/claims/pages/feedback.html.erb

```diff
@@ -0,0 +1,9 @@
+<% content_for :page_title, t(".page_title") %>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l"><%= t(".page_title") %></h1>
+    </div>
+  </div>
+</div>
```

Comment:
Is this page needed? I think usually the 'feedback' link in the phase banner would go directly to the Microsoft Form.

For example, that's what these other services do:

- [Register trainee teachers](https://www.register-trainee-teachers.service.gov.uk/sign-in)
- [Publish teacher training courses](https://www.publish-teacher-training-courses.service.gov.uk/sign-in)

I'm not going to block this PR because it still removes unnecessary code. But I expect you might end up needing to remove this very soon, too.